src/main/java/com/scmp/framework/slackbot/SlackbotService.java

```diff
@@ -0,0 +1,53 @@
+package com.scmp.framework.slackbot;
+
+import com.scmp.framework.context.ApplicationContextProvider;
+import com.scmp.framework.context.FrameworkConfigs;
+import com.scmp.framework.context.RunTimeContext;
+import okhttp3.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.ApplicationContext;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+
+@Component
+public class SlackbotService {
```

Comment:
@benliscmp let's move this class to package com.scmp.framework.services?

Reply:
Moved the class under services package